apps/meteor/server/startup/migrations/v260.ts

```diff
@@ -32,7 +32,7 @@ addMigration({
 			});
 
 			if (updates.length) {
-				Promise.await(VisitorsRaw.col.bulkWrite(updates));
+				Promise.await(LivechatVisitors.col.bulkWrite(updates));
```

Comment:
migrations can be `async`
```suggestion
				await LivechatVisitors.col.bulkWrite(updates);
```

Reply:
C'mon, this mig is from like 4 months ago :trollface: